### PR TITLE
PIM-7760-7761: Get one or several ref entities

### PR DESCRIPTION
**Pull request by NolwennP**
This PR also contains the renaming of the API reference menu (WIP)

TODO:
- Screenshot of the list of the records of one given ref entities: @Delphineray Done ✅

### CODE REVIEW

File: content/rest-api/resources.md [hunk 1]
```diff
@@ -1079,11 +1079,54 @@ Below is the JSON standard format representing a published product. Notice how t
...
+```json
+{
+  "code": "brands",
+  "attributes": {
+    "description": {
+      "labels": {
+        "en_US": "Description",
+        "fr_FR": "Description"
+      },
+      "type": "",
```
Comment (LaurentPetard) [inline content/rest-api/resources.md]: "type": "text" ? (same for "country")

Comment (NolwennP) [reply content/rest-api/resources.md]: My bad. The attributes are not embedded anymore in the ref entity. They will have their own route.

File: content/rest-api/resources.md [hunk 2]
```diff
@@ -1079,11 +1079,35 @@ Below is the JSON standard format representing a published product. Notice how t
...
 
+### Reference entity (3.x and EE only)
+
+Reference entities have been introduced in the PIM in the 3.0 version.
+
+In the Akeneo UI, you can find the reference entities in the `Entities` menu. Below is an example of the `Brand` reference entity with its [records](/documentation/resources.html#reference-entity-record-3x-and-ee-only) in the UI.
+
+![Reference entity](/img/screenshots/v3.0/reference_entity.png)
+
+Below is the JSON standard format representing a reference entity.
```
Comment (ahocquard) [inline content/rest-api/resources.md]:
```suggestion
Below is the JSON format representing a reference entity.
```

File: content/swagger/resources/reference_entities/routes/reference_entities.yaml [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+      examples: {
+        _links: {
+          self: {
+            href: "https://demo.akeneo.com/api/rest/v1/reference-entities?limit=3"
+          },
+          first: {
+            href: "https://demo.akeneo.com/api/rest/v1/reference-entities?limit=3"
+          },
+          next: {
+            href: "https://demo.akeneo.com/api/rest/v1/reference-entities?page=3&search_after=2x055w%3D%3D"
```
Comment (ahocquard) [inline content/swagger/resources/reference_entities/routes/reference_entities.yaml]:
```suggestion
            href: "https://demo.akeneo.com/api/rest/v1/reference-entities?limit=3&search_after=2x055w%3D%3D"
```

File: content/swagger/resources/reference_entities/routes/reference_entities_code.yaml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  x-versions:
+    - "3.0"
+  x-ee: true
+  description: This endpoint allows you to get the information about a given reference entity.
+  parameters:
+    - $ref: '#/parameters/code'
+  responses:
+    200:
+      description: OK
+      x-details: Returns the content of the reference entity in JSON standard format.
```
Comment (ahocquard) [inline content/swagger/resources/reference_entities/routes/reference_entities_code.yaml]:
```suggestion
      x-details: Returns the content of the reference entity in JSON format.
```

File: content/swagger/resources/reference_entities/routes/reference_entities.yaml (hunk 3 above)
Comment (NolwennP) [reply content/swagger/resources/reference_entities/routes/reference_entities.yaml]: Thanks!

**Pull request merged**